src/commands/crashes/upload-mappings.ts

```diff
@@ -0,0 +1,86 @@
+import { AppCenterClient, models } from "../../util/apis";
+import { AppCommand, CommandResult } from "../../util/commandline";
+import { ErrorCodes, failure, success } from "../../util/commandline";
+import { hasArg, help, longName, shortName } from "../../util/commandline";
+import { out } from "../../util/interaction";
+import { inspect } from "util";
+import { DefaultApp } from "../../util/profile";
+import UploadSymbolsHelper from "./lib/symbols-uploading-helper";
+import { SymbolType } from "./lib/symbols-uploading-helper";
+
+import * as Fs from "fs";
+import * as Path from "path";
+
+import * as _ from "lodash";
+
+const debug = require("debug")("appcenter-cli:commands:apps:crashes:upload-mappings");
+
+@help("Upload the Android mappings for the application")
+export default class UploadMappings extends AppCommand {
+  @help("Path to an Android mapping.txt file.")
+  @shortName("m")
+  @longName("mapping")
+  @hasArg
+  public mappingPath: string;
+
+  @help("The version name to associate with the mappings.")
+  @shortName("n")
+  @longName("version-name")
+  @hasArg
+  public versionName: string;
+
+  @help("The version code to associate with the mappings.")
+  @shortName("c")
+  @longName("version-code")
+  @hasArg
+  public versionCode: string;
+
+  public async run(client: AppCenterClient): Promise<CommandResult> {
+    const app: DefaultApp = this.app;
+
+    this.validateParameters();
+
+    this.getStatsForFsPath(this.mappingPath);
+    if (Path.extname(this.mappingPath).toLowerCase() !== ".txt") {
+      throw failure(ErrorCodes.InvalidParameter, `path ${this.mappingPath} does not point to valid mapping.txt file`);
```

Comment:
Can we be more explicit in this message? It'd be helpful to point out that we require file to have `.txt` as extension

Reply:
Yeah, I can see how it might be improvable.